.github/workflows/format.yml

```diff
@@ -10,26 +10,26 @@ on:
       - master
   pull_request:
 jobs:
-  format-and-commit:
-    runs-on: ubuntu-latest
-    name: "Apply All Formatting Rules"
+  format-check:
+    runs-on: "conn-prod-xlarge-runner"
+    name: "Check All Formatting Rules"
     timeout-minutes: 40
     steps:
       - name: Checkout Airbyte
         uses: actions/checkout@v3
         with:
           ref: ${{ github.head_ref }}
           # Important that this is set so that CI checks are triggered again
-          # Without this we would be be forever waiting on required checks to pass
+          # Without this we would be forever waiting on required checks to pass
           token: ${{ secrets.GH_PAT_APPROVINGTON_OCTAVIA }}
 
-      # IMPORTANT! This is nessesary to make sure that a status is reported on the PR
-      # even if the workflow is skipped. If we used github actions filters, the workflow
+      # IMPORTANT! This is necessary to make sure that a status is reported on the PR
+      # even if the workflow is skipped. If we used GitHub Actions filters, the workflow
       # would not be reported as skipped, but instead would be forever pending.
```

Comment:
If we're not comfortable with this I think we should remove the `changed-files` action call and make the airbyte-ci command run on all PRs.
The format command would perform a no-op according the `modified_files` list and report a success status if only markdown files are modified.

Reply:
Just saw this!

@alafanechere I dont think Run on all is the right approach.

If we do that means time to merge a doc only change is unessesarily longer as GHA spin up + Runner spin up + Dagger spin up + format run all add up to > 5mins

Reply:
Sure. But In any case a `GHA spin up + Runner spin up` is still required as the `changed-file` action will run on our self hosted runner.